<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> Error while running shlibload. test case
   </TITLE>
   <LINK REL="Index" HREF="https://mta.openssl.org/pipermail/openssl-users/2019-September/index.html" >
   <LINK REL="made" HREF="mailto:openssl-users%40openssl.org?Subject=Re%3A%20Error%20while%20running%20shlibload.%20test%20case&In-Reply-To=%3C5f753605-eefe-7196-1356-9c8efbe8e3ea%40openssl.org%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="011231.html">
   <LINK REL="Next"  HREF="011232.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>Error while running shlibload. test case</H1>
    <B>Matt Caswell</B> 
    <A HREF="mailto:openssl-users%40openssl.org?Subject=Re%3A%20Error%20while%20running%20shlibload.%20test%20case&In-Reply-To=%3C5f753605-eefe-7196-1356-9c8efbe8e3ea%40openssl.org%3E"
       TITLE="Error while running shlibload. test case">matt at openssl.org
       </A><BR>
    <I>Wed Sep 25 12:34:59 UTC 2019</I>
    <P><UL>
        <LI>Previous message: <A HREF="011231.html">Error while running shlibload. test case
</A></li>
        <LI>Next message: <A HREF="011232.html">Working inside X509_STORE_CTX using verification callbacks
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#11234">[ date ]</a>
              <a href="thread.html#11234">[ thread ]</a>
              <a href="subject.html#11234">[ subject ]</a>
              <a href="author.html#11234">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>

On 25/09/2019 12:18, shiva kumar wrote:
&gt;<i> Hi,
</I>&gt;<i> I'm getting error while running&#160;test case&#160; &quot;90-test_shlibload.t&quot; in openssl
</I>&gt;<i> 1.1.1c make test.
</I>&gt;<i> 
</I>&gt;<i> ../test/recipes/90-test_shlibload.t (Wstat: 1024 Tests: 10 Failed: 4)
</I>&gt;<i> &#160; Failed tests: &#160;2, 4, 6, 8
</I>&gt;<i> 
</I>&gt;<i> &#160;in file&#160; test/recipes/90-test_shlibload.t&#160; &#160;
</I>&gt;<i> *ok(check_atexit($fh));*&#160;&#160;
</I>&gt;<i> 
</I>&gt;<i> when i checked it is the above function call, which is throwing error.
</I>&gt;<i> can anyone please tell me why is this happening?
</I>
When OpenSSL is unloaded it is supposed to do some cleanup via an &quot;atexit&quot;
handler. This test checks that the handler was called and therefore that the
cleanup occurred. For some reason it didn't for you.

What platform are you running this on and how did you build OpenSSL?

Matt

</PRE>







<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="011231.html">Error while running shlibload. test case
</A></li>
	<LI>Next message: <A HREF="011232.html">Working inside X509_STORE_CTX using verification callbacks
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#11234">[ date ]</a>
              <a href="thread.html#11234">[ thread ]</a>
              <a href="subject.html#11234">[ subject ]</a>
              <a href="author.html#11234">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="../../../mailman/listinfo/openssl-users.html">More information about the openssl-users
mailing list</a><br>
</body></html>
